### ISSUE TITLE
Change Windows Release exe name

### ISSUE THREAD

**Issue by Murmele** [Murmele/Gittyup#147]
@exactly-one-kas 

After every release the following files must be updated which might be error prone. What do you think about removing the version number of the exe?

- Readme.md
- index.md

**Comment by alerque** [comment]
Release artifacts should always have the version in the name. Without it lots of mirroring/caching/packaging mechanisms will break. You want to be able to have more than one in a directory as well. Just change the readme to link to the release, not the exact file. You can also use the `latest` keyword to link to the latest release without knowing the tag, which is useful for README and other places you don't want to bump on updates.

**Comment by Murmele** [comment]
@alerque thank you for your hint. I will checkout if it is possible to reduce.

**Comment by Murmele** [comment]
> 

But using the latest keyword does not help, because you have to change in the Readme also the executable name so you have to change always there

**Comment by exactly-one-kas** [comment]
Maybe we could let the markdown files be generated by CMake and have them be published onto `gh-pages` via GitHub Actions on pushes on `master` using https://github.com/marketplace/actions/github-push?

**Comment by Murmele** [comment]
Yes I thought about something like that too.

I set up yesterday that the github page will be created and published as soon as a new gittyup tag will be set. So for changing index.md would work

but for the Readme.md it must be committed separately (For me this is not a problem or what do you think?).

I thought also creating the appdata.xml with cmake so the release notes have to be written only once. But this is another story ^^.
I like also to have information at one place and auto generating other files
